e2e/provision/galaxy-requirements.yml

```diff
@@ -24,7 +24,7 @@ roles:
 
 collections:
   - name: community.general
-    version: 9.1.0
+    version: 8.6.2
```

Comment:
Curious as to why we are downgrading here?

Reply:
We are getting a 403 error on the 9.1.0 version.

Reply:
@liamfallon Thanks